dawn/src/dawn/Optimizer/StatementMapper.cpp

```diff
@@ -185,7 +185,10 @@ void StatementMapper::visit(const std::shared_ptr<iir::UnaryOperator>& expr) {
 }
 
 void StatementMapper::visit(const std::shared_ptr<iir::ReductionOverNeighborExpr>& expr) {
-  DAWN_ASSERT_MSG(0, "ReductionOverNeighborExpr not allowed in this context");
+  DAWN_ASSERT(initializedWithBlockStmt_);
+
+  for(auto& s : expr->getChildren())
```

Comment:
we usually also put `{}` around single statement for loops. Worth for consistency?

Reply:
see above